Frontend/ui-common/src/Engineering/Helpers/JBridge.js

```diff
@@ -310,7 +310,7 @@ export const getKeyInSharedPrefKeys = function (key) {
   return JBridge.getFromSharedPrefs(key);
 };
 
-export const checkAndAskNotificationPermission = function (unit) {
+export const checkAndAskNotificationPermission = function (shouldAlwaysAsk) {
```

Comment:
what's the requirement of `shouldAlwaysAsk` if we already created another function to check for permission?